src/match/queue.ts

```diff
@@ -0,0 +1,48 @@
+import { randomUUID } from 'crypto';
+
+export const subscribe = () => {};
+type Item = {
+  id: string;
+  joinAt: number;
```

Comment:
Hmm, is there a typescript type for Date objects?

Reply:
joinAt is a timestamp , so I use number here

Reply:
I mean this:

<img width="958" alt="image" src="https://user-images.githubusercontent.com/42633156/233845907-4a26d1e6-5b2a-40cd-8648-33d9e0cc1432.png">

https://www.typescriptlang.org/docs/handbook/2/basic-types.html#explicit-types